### ISSUE TITLE
0013640:
Search for recepients brings irrelevant results

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#6746]
**Reported by apvit on 24 Nov 2017 10:39**

**Version:** 2017.08.9 Community Edition

The search for recepients in felamimail, brings irrelevant results.
See attached file. Mail addresses have been removed for confidentiality reasons. The search text does not occur in any of them, except of the searched address.

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 11 Jan 2018 14:01**

hi,

do you have activated the &quot;extended search&quot; option in your user preferences (see screenshot)?

this extends the search to all related records of the contacts you have access to.

**Comment by Gloirin** [comment]
**Comment posted by apvit on 15 Jan 2018 07:17**

Extended search was activated.
But we found out, that this is not the problem.

The problem ist, that the search function also looks into field description. 
So, when the search-string occurs in the description field, the search is positive an the name of the contact is being displayed in the list.

We sometimes import addresses, where certain names are in the field &quot;description&quot;.
We solved the problem temporarily, by eliminating the names via sql.
But it would be better, if the search function would search only in the name-fields and eventually in email-address-fields.
